engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-errata-modal.html

```diff
@@ -3,9 +3,9 @@ <h4 data-block="modal-header">Content Host Errata Management</h4>
 
   <div data-block="modal-body">
     <form id="errataActionForm" name="errataActionForm" class="form" method="post" action="/katello/remote_execution">
-      <input type="hidden" name="name" ng-value="errataActionFormValues.errata"/>
       <input type="hidden" name="remote_action" ng-value="errataActionFormValues.remoteAction"/>
       <input type="hidden" name="host_ids" ng-value="errataActionFormValues.hostIds"/>
+      <input type="hidden" name="bulk_errata_ids" ng-value="errataActionFormValues.bulkErrataIds"/>
```

Comment:
i think this fixes bulk_errata_ids, but still leaves host_ids broken (returning just a comma seperated list).  can you fix that too in the same way? (probably also have to update the controller to not split the incoming string of ids)

Reply:
Certainly!